### PR TITLE
Visitor can navigate through the website 

### PR DESCRIPTION
**Pull request by johnnysee**
Link to PT: https://www.pivotaltracker.com/story/show/179886803

## User Story

```
As a visitor,
In order to view the home page
I want to be able to navigate through the website
```

## Tasks

- Cypress tests for different tabs
- Create Header & Footer components
- Create Home page and set it to default route with Link react-router-dom
- Installed Semantic-UI for styling Header & Footer...
- Added styles.css to always put footer in the bottom

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+import { Switch, Route, BrowserRouter } from "react-router-dom";
 
 const App = () => {
   return (
-    <div>
-      <h1>Hello world!</h1>
-    </div>
-  )
-}
+    <>
```
Comment (tochman) [inline src/App.jsx]: No need for this fragment

File: src/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const Footer = () => {
+  return (
+    <>
```
Comment (tochman) [inline src/Footer.jsx]: No need for this fragment

File: src/HomePage.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Container } from "semantic-ui-react";
+
+const HomePage = () => {
+  return (
+    <>
```
Comment (tochman) [inline src/HomePage.jsx]: Same here

File: public/index.html [hunk 4]
```diff
@@ -12,6 +12,7 @@
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>NIP</title>
+    <link rel="stylesheet" href="styles.css" />
```
Comment (tochman) [inline public/index.html]: Not very React-ish

File: public/styles.css [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  position: absolute;
+  text-align: center;
+  bottom: 0;
+  width: 100%;
+  height: 2.5rem;
+}
+
+footer p {
+  color: #ccc;
+}
```
Comment (tochman) [inline public/styles.css]: I would place this file in `src` and import it in `index.js`

**Review by tochman — CHANGES_REQUESTED** [review]

**Review by maxarvid — COMMENTED** [review]
LGTM!

File: src/App.css [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+  position: absolute;
+  text-align: center;
+  bottom: 0;
+  width: 100%;
+  height: 2.5rem;
+}
+
+footer p {
+  color: #ccc;
+}
```
Comment (tochman) [inline src/App.css]: So, if you call this file `App.css` then you kind of scope the CSS that this file contains to only be used in the `App` component. And it does not, right?  So you have 2 choices in my opinion. You either call it `index.css` and import it in `index.js` (as you do below), or you call it `Footer.css` and import it in the `Footer` component. 

I would suggest the first variant but you need to form your own opinion on this though.

File: src/index.js [hunk 7]
```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App";
+import "semantic-ui-css/semantic.min.css";
+import "./App.css";
```
Comment (tochman) [inline src/index.js]: See comment on the css file...

**Review by tochman — CHANGES_REQUESTED** [review]

**Review by AugustEnhager — COMMENTED** [review]
Other than Thomas comment on css file, it looks good!

File: src/App.css (hunk 6 above)
Comment (johnnysee) [reply src/App.css]: Done, renamed App.css to index.css.

Also created a components and styles folders to store respectively files in there.

Thanks.

**Review by tochman — APPROVED** [review]

**Pull request merged**